exporter/awsemfexporter/datapoint_test.go

```diff
@@ -347,6 +374,86 @@ func TestCalculateDeltaDatapoints_HistogramDataPointSlice(t *testing.T) {
 
 }
 
+func TestCalculateDeltaDatapoints_ExponentialHistogramDataPointSlice(t *testing.T) {
+	deltaMetricMetadata := generateDeltaMetricMetadata(false, "foo", false)
+
+	testCases := []struct {
```

Comment:
More test cases would be good here.  What about different scales?  Different attributes?  Different sums?  The output of these fields could be hard coded and these tests would still pass.

Reply:
Agree. More tests are added to verify those cases.